SF iOS/SF iOS/Assets.xcassets/button - subscribed.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "button - subscribed.pdf",
```

Comment:
I’m worried these buttons are really clunky looking and want to tweak them a bit to be more beautiful.

Reply:
I'm open to it.